src/App.tsx

```diff
@@ -1,24 +1,35 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodosList } from './components/TodosList/TodosList';
 
-const USER_ID = 0;
+const USER_ID = 6937;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
```

Comment:
if you will have differrent error you should use here string and pass it in line 59